src/message.h

```diff
@@ -35,6 +35,7 @@ extern void err(const char *fmt, ...) PRINTFLIKE(1, 2);
 extern void err_full(const char *file,int line,const char *fmt, ...) PRINTFLIKE(3, 4);
 extern void term(const char *fmt, ...) PRINTFLIKE(1, 2);
 void initWarningFormat();
+void warn_flush(void);
```

Comment:
same here `(void)` is equal to `()`

Reply:
I'm more from the old sort, I like the clarity.
I'll change it for consistency reasons.

Reply:
It is also the recommended syntax in the C++ FAQ: http://www.dietmar-kuehl.de/mirror/c++-faq/newbie.html#faq-29.4